frontend/document/koptinterface.lua

```diff
@@ -668,6 +668,9 @@ function KoptInterface:getNativeTextBoxesFromScratch(doc, pageno)
         local page = doc._document:openPage(pageno)
         page:getPagePix(kc)
         local boxes, nr_word = kc:getNativeWordBoxes("src", 0, 0, page_size.w, page_size.h)
+        if not boxes then
+            return
+        end
```

Comment:
This doesn't look quite the same as what you linked. See the page:close() and the kc:free() a few lines down.

Reply:
Yes, thanks, corrected. Now it does not kill but still no highlight shown
```
rendering page:0,0,4908,6382
Warning in boxaSelectBySize: boxas is empty
Warning in boxaSelectBySize: boxas is empty
Error in boxaSort2d: boxas is empty
Error in pixaSort2dByIndex: naindex not defined
Error in pixaaFlattenToPixa: paa not defined
Error in pixaGetBoxa: pixa not defined

```